pygmt/helpers/__init__.py

```diff
@@ -19,6 +19,7 @@
     args_in_kwargs,
     build_arg_list,
     build_arg_string,
+    check_encoding,
```

Comment:
Thoughts on using `_check_encoding` to keep this function more private? I know we don't document `pygmt.helpers.utils` in the API docs, but want to avoid users from thinking that this function is somewhat public if there's no leading underscore.

Reply:
Done in 6728856a90ea66a96a543c78981fbb8fad342bce.